src/Dflydev/FigCookies/Cookies.php

```diff
@@ -71,28 +72,26 @@ public function without(string $name) : Cookies
     /**
      * Render Cookies into a Request.
      */
-    public function renderIntoCookieHeader(RequestInterface $request) : RequestInterface
+    public function renderIntoCookieHeader(RequestInterface $request): RequestInterface
     {
         $cookieString = implode('; ', $this->cookies);
 
-        $request = $request->withHeader(static::COOKIE_HEADER, $cookieString);
+        $request = $request->withHeader(self::COOKIE_HEADER, $cookieString);
 
         return $request;
     }
 
     /**
      * Create Cookies from a Cookie header value string.
-     *
-     * @return static
      */
-    public static function fromCookieString(string $string) : self
+    public static function fromCookieString(string $string): self
     {
         return new static(Cookie::listFromCookieString($string));
     }
 
-    public static function fromRequest(RequestInterface $request) : Cookies
+    public static function fromRequest(RequestInterface $request): Cookies
     {
-        $cookieString = $request->getHeaderLine(static::COOKIE_HEADER);
+        $cookieString = $request->getHeaderLine(self::COOKIE_HEADER);
```

Comment:
changes are low (http spec), but also this constant could be overridden

Reply:
I think I've completely underestimated the `static::` vs `self::` implications in a few places... Is doing `static::` bad here for some reason?